presto-bigquery/src/main/java/io/prestosql/plugin/bigquery/BigQueryConnectorModule.java

```diff
@@ -94,6 +95,6 @@ static String calculateBillingProjectId(Optional<String> configParentProjectId,
             return ((ServiceAccountCredentials) credentials.get()).getProjectId();
         }
         // 3. No configuration was provided, so get the default from the environment
-        return BigQueryOptions.getDefaultProjectId();
+        return ServiceOptions.getDefaultProjectId();
```

Comment:
hm... `BigQueryOptions` seemed more appropriate (i don;t know the design behind BigQueryOptions vs generic ServiceOptions though)

Reply:
I'll drop this commit